sros2/api/__init__.py

```diff
@@ -356,16 +356,17 @@ def create_permission_file(path, name, domain_id, permissions_dict):
             'list_parameters',
             'describe_parameters',
         ]
-        for key, value in service_topic_prefixes.items():
-            if key == 'Request':
+        for service_direction, topic_prefix in service_topic_prefixes.items():
+            if service_direction == 'Request':
```

Comment:
Thanks for looking into that. I couldnt find a "good" name when I refactored so I didnt change it.

I don't think that "service_direction" is what we mean here. For example, a service server would subscribe to a `Request` topic.

I made another attempt at renaming the variable in https://github.com/ros2/sros2/pull/45/commits/70bc265690e547a03600538436ff4d236ce740c5 trying to match the "topic_prefix" variable introduced here. Feedback welcome

Reply:
yep, good point, that name looks good!